lobsterpy/cohp/analyze.py

```diff
@@ -603,7 +614,7 @@ def _get_bond_resolved_data_stats(self, orb_resolved_bond_data: dict):
         Returns:
             dict with orbital data stats the site for relevant orbitals, e.g.
             {'orbital_summary_stats': {'max_bonding_contribution': {'2s-3s': 0.68},
-            'max_antibonding_contribution': {'2s-2pz': 0.36}}}
+            'max_antibonding_contribution': {'2s-2p': 0.36}}}

```

Comment:
```suggestion
            'max_antibonding_contribution': {'2s-2p': 0.36}}}

```

Why does this stay the same depite "sum"?

Reply:
I did not put exact values there. This is just the random number in doc-strings to indicate what to expect 😅

Reply:
Ah. Doc string. Sorry!

Reply:
No worries. But I will update it to a reasonable number to avoid confusion